TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -0,0 +1,207 @@
+using System;
+using System.Collections.Generic;
+using ColossalFramework;
+using ColossalFramework.Globalization;
+using ColossalFramework.PlatformServices;
+using ColossalFramework.Plugins;
+using ColossalFramework.UI;
+using CSUtil.Commons;
+using UnityEngine;
+
+namespace TrafficManager.UI {
+    public class IncompatibleModsPanel : UIPanel {
+        public UILabel title;
+        public UIButton closeButton;
+        public UISprite warningIcon;
+        private static IncompatibleModsPanel _instance;
+
+        public static IncompatibleModsPanel Instance {
+            get {
+                if (_instance == null) {
+                    _instance = (UIView.GetAView().AddUIComponent(typeof(IncompatibleModsPanel)) as IncompatibleModsPanel);
+                }
+
+                return _instance;
+            }
+        }
+
+        public Dictionary<ulong, string> IncompatibleMods { get; set; }
+
+        public void Initialize() {
+            Log._Debug("IncompatibleModsPanel initialize");
+            isVisible = true;
+
+            backgroundSprite = "MenuPanel3";
```

Comment:
Change sprite at this line for different background

Reply:
iterating various panel bgs, brb...

Reply:
* "InfoDisplay" -- solid black
* "MenuContainer" -- midnight blue gradient
* "MenuPanel2" -- solid navy blue
* "UnlockingPanel2" -- sort of a mix between previous two

IMO **```UnlockingPanel2```** is the best one - it's distinct, so users won't mistake it for occasional "New DLC Features" panel, and it's fully opaque  so very easy to read the contents.

Reply:
Yup, ```UnlockingPanel2``` looks great